localstack/services/apigateway/provider.py

```diff
@@ -508,6 +520,14 @@ def update_method(
         applicable_patch_operations = []
         modifying_auth_type = False
         modified_authorizer_id = False
+        had_req_params = (
+            isinstance(moto_method.request_parameters, dict)
+            and len(moto_method.request_parameters) > 0
+        )
+        had_req_models = (
+            isinstance(moto_method.request_models, dict) and len(moto_method.request_models) > 0
```

Comment:
Non-empty collections are "truthy" I don't necessary think this makes it more readable.